varia/functions.php

```diff
@@ -268,6 +294,7 @@ function varia_scripts() {
 			array( 'css-vars-ponyfill' ),
 			'1.0'
 		);
+		wp_enqueue_style( 'varia-print-style', get_template_directory_uri() . '/ie.css', array(), wp_get_theme()->get( 'Version' ), 'print' );
```

Comment:
Should this style handle be renamed and not refer to `print` styles?

```suggestion
		wp_enqueue_style( 'varia-ie-styles', get_template_directory_uri() . '/ie.css', array(), wp_get_theme()->get( 'Version' ) );
```